docs-source/content/userguide/cicd/choose-an-approach.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Choose an Approach"
+date: 2019-04-11T13:36:57-04:00
+draft: false
+weight: 3
+decsription: "How to choose an approach."
+---
+
+Let's review what we have discussed and talk about when we might want to use 
+various approaches.  We can start by asking ourselves questions like these:
+
+
+- *Can you make the desired change with a configuration override?*  
+  The WebLogic Kubernetes Operator allows you to inject a number of [configuration 
+  overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}) 
+  into your pods at startup time.  This allows you to use the same image for multiple 
```

Comment:
at startup time. --> before starting any servers in the domain.   (E.g. try make it clear taht it's not something useful for rolling upgrades -- or maybe just flat out state that its not suited for rolling upgrades.)

Reply:
done